governance.Jenkinsfile

```diff
@@ -1,12 +1,15 @@
 #!/usr/bin/env groovy
 def label = "kyma-${UUID.randomUUID().toString()}"
+def isMaster = params.GIT_BRANCH == 'master'
+def repositoryName = "examples"
 
 echo """
 ********************************
 Job started with the following parameters:
 GIT_REVISION=${params.GIT_REVISION}
 GIT_BRANCH=${params.GIT_BRANCH}
 APP_VERSION=${params.APP_VERSION}
+TRIGGER_FULL_VALIDATION=${params.TRIGGER_FULL_VALIDATION}
```

Comment:
Is this param only intended for manual configuration as it is not added to orchestrator?

Reply:
It is already added to `orchestrator`, please check Jenkins and `seedJob`.